### PR TITLE
Add WSFC section to main readme.md

### PR DESCRIPTION
**Pull request by ning-yang**
Detail user guide link will be added once it is available.

### CODE REVIEW

**Comment by ning-yang** [comment]
@adjackura let me know if it looks good to you. I cannot add reviewer for some reason.

File: README.md [hunk 1]
```diff
@@ -38,6 +38,18 @@ The agent uses IP forwarding metadata to setup or remove IP routes.
...
 
+#### Windows Failover Cluster Support
+
+The agent can monitor the active node in the [Windows Failover Cluster](https://technet.microsoft.com/en-us/library/cc770737(v=ws.11).aspx) and coordinate with GCP [Internal Load Balancer](https://cloud.google.com/compute/docs/load-balancing/internal/) to forward all cluster traffic to the expected node.
+
+The following fields on instance metadata or instance_configs.cfg can control the behavior:
+
+* `enable-wsfc`: If set to true, all IP forwarding info will be ignored and agent will start responding to the health check port. Default false.
+* `wsfc-agent-port`: The port which health checker from load balancer will use to determine the active node. Default 59998.
+* `wsfc-addrs`: A comma separated IP address. This is an advanced setting to enable user have both normal forwarding IPs and cluster IPs on the same instance. If set, agent will only skip-auto configuring IPs in the list. Default empty.
```
Comment (adjackura) [inline README.md]: A comma separated list of IP address

File: README.md [hunk 2]
```diff
@@ -38,6 +38,18 @@ The agent uses IP forwarding metadata to setup or remove IP routes.
...
 *   Only IPv4 IP addresses are currently supported.
 
+#### Windows Failover Cluster Support
+
+The agent can monitor the active node in the [Windows Failover Cluster](https://technet.microsoft.com/en-us/library/cc770737(v=ws.11).aspx) and coordinate with GCP [Internal Load Balancer](https://cloud.google.com/compute/docs/load-balancing/internal/) to forward all cluster traffic to the expected node.
+
+The following fields on instance metadata or instance_configs.cfg can control the behavior:
+
+* `enable-wsfc`: If set to true, all IP forwarding info will be ignored and agent will start responding to the health check port. Default false.
+* `wsfc-agent-port`: The port which health checker from load balancer will use to determine the active node. Default 59998.
```
Comment (adjackura) [inline README.md]: This description is a bit confusing maybe "The port which the agent will respond to health checks."  ?

Comment (ning-yang) [inline README.md]: done

File: README.md (hunk 1 above)
Comment (ning-yang) [inline README.md]: updated.

File: README.md [hunk 3]
```diff
@@ -38,6 +38,18 @@ The agent uses IP forwarding metadata to setup or remove IP routes.
 
 *   Only IPv4 IP addresses are currently supported.
 
+#### Windows Failover Cluster Support
```
Comment (adjackura) [inline README.md]: Add to the table of contents above

Comment (ning-yang) [inline README.md]: k!

Comment (ning-yang) [inline README.md]: done

**Pull request merged**